laterpay/application/Controller/Frontend/Post.php

```diff
@@ -105,6 +113,17 @@ public function ajax_redeem_voucher_code( LaterPay_Core_Event $event ) {
         $code       = sanitize_text_field( $_GET['code'] ); // phpcs:ignore
         $code_data  = LaterPay_Helper_Voucher::check_voucher_code( $code );
         $lp_post_id = intval( $_GET['lp_post_id'] ); // phpcs:ignore
+        $type       = filter_input( INPUT_GET, 'type', FILTER_SANITIZE_STRING );
+        $type       = sanitize_text_field( $type );
```

Comment:
Do we need additional sanitization? `filter_input` should be enough in this case? There is no form submission and user input won't be used directly we have set of allowed types i.e `'timepass', 'subscription'`

Reply:
That won't harm to add additional sanitation. Also yes there is not form submission. But there are still chance that some advance user can change the value by browser's developer tool.

Reply:
Additional sanitation doesn't but unnecessary usage is overkill, `$type` isn't used for storage anywhere but only for comparison, I don't have an issue with the sanitization, it's just that it wont be of any use.

Reply:
done.